docs/networks/network-architecture/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Network Architecture
+sidebar_position: 1
+---
+
+Flow has pioneered a new paradigm of multi-node architecture that solves the core problem of today’s blockchains.
+The result is a scalable, decentralized, and secure network which ensures user safety and long-term sustainability.
+
+To better understand the architecture, lets first understand the problems with the current blockchain. Then lets look at how Flow multi-node architecture solves these problems.
+
+# What are the three biggest problems with blockchains?
```

Comment:
What are the biggest problems solved by Flow's Multi-node Architecture?

Reply:
made this change